core/src/main/java/com/github/mizool/core/rest/errorhandling/ErrorHandlingBehaviorCatalog.java

```diff
@@ -10,13 +26,13 @@
 import com.google.common.collect.ImmutableMap;
 
 @Singleton
-class ErrorHandlingCatalog
+class ErrorHandlingBehaviorCatalog
```

Comment:
Speaking of "spring cleanup": while you're at it, please also incorporate #147 in this PR.